src/Tizen.NUI/src/internal/FrameBroker/DefaultFrameBroker.cs

```diff
@@ -63,20 +69,65 @@ protected override void OnFrameResumed(FrameData frame)
             StartAnimation();
         }
 
-        private void PlayAnimateTo(FrameData frame, TransitionAnimation animation)
+        protected override void OnFramePaused()
+        {
+            base.OnFramePaused();
+            animation?.Stop();
+
+            ResetImage();
+
+            isAnimating = false;
+        }
+
+        private bool Timer_Tick(object source, Timer.TickEventArgs e)
```

Comment:
from next time, please do not use "_"
please use pascal case.

Reply:
unused function. I've removed it.